src/state/opamEnv.ml

```diff
@@ -725,6 +718,12 @@ let setup_interactive root ~dot_profile shell =
     (OpamConsole.colorise `cyan @@ OpamFilename.prettify dot_profile)
     (OpamConsole.colorise `bold @@ source root ~shell (init_file shell))
     (OpamConsole.colorise `bold @@ "eval $(opam env)");
+  let eval_env =
+    OpamConsole.confirm ~default:false
+      "Additionally, a hook can be added to your shell to ensure that \
+       the opam environement remains in sync. Add a hook?"
```

Comment:
There's one too many e's in environment

Reply:
(or one too few `n`s ;) )
more seriously, shouldn't that be even later, after the `"Do you want to modify"` question ?
In practice, to the user, this question makes little sense if the answer to the other one is `no`, since they won't source the file anyway (they are more likely to just run `eval $(opam env)`) ; so it should probably only be asked if the other answer was `y` (or `f`).

Reply:
Ok to have it after the "Do you want to modify" question, but if user doesn't want to modify it's `.profile` file, it does not mean that he won't use `init` scripts (it's my case). Agree that it's for non-beginner user...